<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="contenedor_inputs">
     <item row="1" column="0">
      <widget class="QLabel" name="labelB">
       <property name="text">
        <string>B:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelA">
       <property name="text">
        <string>A:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputA"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inputB"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Calculos</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="chSumar">
       <property name="text">
        <string>Sumar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chRestar">
       <property name="text">
        <string>Restar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chMultiplicar">
       <property name="text">
        <string>Multiplicar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chDividir">
       <property name="text">
        <string>Dividir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chParImpar">
       <property name="text">
        <string>Par / Impar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chPosCerNeg">
       <property name="text">
        <string>Positivo / Cero / Negativo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="calcular">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
